artifacts/definitions/Linux/Events/ExecutableFiles.yaml

```diff
@@ -0,0 +1,66 @@
+name: Linux.Events.ExecutableFiles
+description: |
+  This artifact collects Every new executable file in /tmp or login
+  directory from the Linux kernel.
+
+  This artifact relies on the presence of `auditctl` usually included
+  in the auditd package. On Ubuntu you can install it using:
+
+  ```
+  apt-get install auditd
+  ```
+
+  On SUSE operating systems you can install it using:
+  ```
+  zypper install audit
+  ```
+
+precondition: SELECT OS From info() where OS = 'linux'
+
+type: CLIENT_EVENT
+
+required_permissions:
+  - EXECVE
+
+parameters:
+  - name: pathToAugenrules 
+    default: /sbin/augenrules
+    description: We depend on augenrules to install the newly added rules.
+
+sources:
+  - query: |
+     // Install the auditd rule if possible.
+     // Remove default filter if any
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_clear_default.rules ]; then echo '-d task,never -i' > /etc/audit/rules.d/vrr_clear_default.rules; chmod 0640 /etc/audit/rules.d/vrr_clear_default.rules; " + pathToAugenrules + " --load; fi"])
+
+     // add rule to monitor new executable file 
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_exec_bit_b64.rule ]; then echo '-a always,exit -F arch=b64 -S fchmodat -F a2&73 -k vrr_exec_bit_addition' > /etc/audit/rules.d/vrr_exec_bit_b64.rules; chmod 0640 /etc/audit/rules.d/vrr_exec_bit_b64.rules; " + pathToAugenrules + " --load; fi"])
+
+     // add rule to monitor new executable file 
+     LET _ <= SELECT * FROM execve(argv=["/bin/bash", "-c", "if [ ! -f /etc/audit/rules.d/vrr_exec_bit_b32.rule ]; then echo '-a always,exit -F arch=b32 -S fchmodat -F a2&73 -k vrr_exec_bit_addition' > /etc/audit/rules.d/vrr_exec_bit_b32.rules; chmod 0640 /etc/audit/rules.d/vrr_exec_bit_b32.rules; " + pathToAugenrules + " --load; fi"])
+
```

Comment:
I think we should combine the related rules into the same file. i.e. vrr_exec_bit.rules